src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.Session.cs

```diff
@@ -464,21 +464,19 @@ private async Task DebugVerifyNoErrorsAsync(MutableConflictResolution conflictRe
 
             private async Task<ImmutableHashSet<ISymbol>?> GetNonConflictSymbolsAsync(Project currentProject)
             {
-                if (_nonConflictSymbols == null)
+                if (_nonConflictSymbolKeys.IsDefault)
                     return null;
 
                 var compilation = await currentProject.GetRequiredCompilationAsync(_cancellationToken).ConfigureAwait(false);
                 return ImmutableHashSet.CreateRange(
-                    _nonConflictSymbols.Select(s => s.GetSymbolKey().Resolve(compilation).GetAnySymbol()).WhereNotNull());
+                    _nonConflictSymbolKeys.Select(s => s.Resolve(compilation).GetAnySymbol()).WhereNotNull());
```

Comment:
we used to serialize the symbols over, only to get symbol keys, which rename would tehn look up in the current compilation it was forking as it does its multiphase conflict resolution.  Now, we just send the symbol key over since we're going to have to convert to that anyways.